src/less/variables-payex.less

```diff
@@ -90,6 +90,9 @@
 @link-hover-color: @brand-primary;
 @link-color: @btn-primary-bg;
 
+/* Button active border color */
+@btn-active-border-color: #3c3c3c;
+
```

Comment:
as mentioned before in another comment in "src/less/components/payex/button.less",

If possible reuse an existing variable (e.g. `--brand-secondary` ?). Especially since this variable does not exist on the swedbankpay variables. So it's more probable the use of this variable will get forgotten.

If this being said you believe it to be necessary to have this new variable then please use custom-properties instead of LESS variables 😉